paderbox/utils/nested.py

```diff
@@ -400,3 +413,196 @@ def squeeze_nested(orig):
         if squeezed and all([orig[key] == orig[keys[0]] for key in keys]):
             return orig[keys[0]]
     return orig
+
+
+def _get_by_path(
+        d: Union[Mapping, Sequence],
+        path: Tuple[Any, ...],
+        broadcast: bool = False
+):
+    """ Helper function for `get_by_path` and `set_by_path`. """
+    for k in path:
+        try:
+            d = d[k]
```

Comment:
For broadcast you have to catch much more cases. The exception can have any type.